app/samples/phase_detection/src/flight.h

```diff
@@ -0,0 +1,121 @@
+#pragma once
+#include <array>
+#include <cstdint>
+#include <f_core/flight/c_phase_controller.h>
+
+/**
+ * Flight Events for a fictional flight
+ */
+enum Events : uint8_t { PadReady, Boost, Coast, Noseover, MainChute, GroundHit, CamerasOff, NumEvents };
+inline constexpr std::array<const char *, Events::NumEvents> eventNames = {
+    "PadReady", "Boost", "Coast", "Noseover", "MainChute", "GroundHit", "CamerasOff",
+};
+
+/**
+ * Sources of flight events
+ */
+enum Sources : uint8_t {
+    IMU1,
+    Barom1,
+    NoseoverLockout,
+    Boost2CoastTimer,
+    Noseover2MainTimer,
+    FullFlightTimer,
+    ExtraCameraTimer,
+    NumSources
+};
+inline constexpr std::array<const char *, Sources::NumSources> sourceNames = {
+    "IMU 1 (LSM6DSL)",     "Barom 1 (BME280)",  "Noseover Lockout",  "Boost2Coast Timer",
+    "Noseover2Main Timer", "Full Flight Timer", "ExtraCamera Timer",
+};
+
+inline constexpr std::size_t num_timer_events = 5;
```

Comment:
Bad style

Reply:
In the capitalization sense? I thought we were rocking snake case for vars. Or would we want MACRO_CASE for this where this is effectively just a typed macro.